src/services/content.js

```diff
@@ -5,22 +5,21 @@
 import {
   fetchByRailContentIds,
   fetchMetadata,
-  fetchRecent,
   fetchTabData,
   fetchNewReleases,
   fetchUpcomingEvents,
   fetchScheduledReleases,
   fetchReturning,
   fetchLeaving, fetchScheduledAndNewReleases, fetchContentRows
 } from './sanity.js'
+import {getRecentTypesForPage} from '../contentTypeConfig.js'
 import {TabResponseType, Tabs, capitalizeFirstLetter} from '../contentMetaData.js'
-import {fetchHandler} from "./railcontent";
+import {fetchRecent} from "./railcontent";
 import {recommendations, rankCategories, rankItems} from "./recommendations";
 import {addContextToContent} from "./contentAggregator.js";
 
-
 export async function getLessonContentRows (brand='drumeo', pageName = 'lessons') {
```

Comment:
These should not be defaulted and should always be populated by the caller
```suggestion
export async function getLessonContentRows (brand, pageName) {
```